src/activities/guesscount-summer/GuesscountSummer.qml

```diff
@@ -0,0 +1,343 @@
+ *   RAHUL YADAV <rahulyadav170923@gmail.com> (Qt Quick port)
+ *
+ *   This program is free software; you can redistribute it and/or modify
+ *   it under the terms of the GNU General Public License as published by
+ *   the Free Software Foundation; either version 3 of the License, or
+ *   (at your option) any later version.
+ *
+ *   This program is distributed in the hope that it will be useful,
+ *   but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *   MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *   GNU General Public License for more details.
+ *
+ *   You should have received a copy of the GNU General Public License
+ *   along with this program; if not, see <http://www.gnu.org/licenses/>.
+ */
+import QtQuick 2.1
+
+import "../../core"
+import "guesscount-summer.js" as Activity
+
+ActivityBase {
+    id: activity
+
+    onStart: focus = true
+    onStop: {}
+    property bool needRestart: true
+
+    pageComponent: Rectangle {
+        id: background
+        anchors.fill: parent
+        color: "#ABCDEF"
+        signal start
+        signal stop
+        Component.onCompleted: {
+            dialogActivityConfig.getInitialConfiguration()
+            activity.start.connect(start)
+            activity.stop.connect(stop)
+        }
+
+        // Add here the QML items you need to access in javascript
+        QtObject {
+            id: items
+            property Item main: activity.main
+            property alias background: background
+            property alias bar: bar
+            property alias bonus: bonus
+            property int sublevel: 0
+            property alias operand_row : operand_row
+            property  var data
+            property int result: data[sublevel-1][1]
+            property alias timer: timer
+            property alias dialog: dialog
+            property GCAudio audioEffects: activity.audioEffects
+            property bool solved
+            property bool levelchanged : false
+            property alias parser: parser
+            property var level_arr
+            property alias load: dialogActivityConfig
+            property alias operators: items.level_arr
+            property string mode
+            property int currentlevel
+        }
+
+        onStart:  if (activity.needRestart) {
+                      Activity.start(items);
+                      activity.needRestart = false;
+                  } else
+                      Activity.initLevel();
+        onStop: { Activity.stop() }
+
+        JsonParser {
+            id: parser
+            onError: console.error("Guesscount: Error parsing JSON: " + msg);
+        }
+
+        Loader {
+            id: admin
+            active: false
+            sourceComponent: Column {
+                spacing: 10
+                width: parent.width
+                height: parent.height
+
+                Repeater{
+                    id:levels
+                    model: 6
```

Comment:
if I'm not wrong, it should be Activity.numberOfLevels instead of having a hardcoded value